libs/wire-api/src/Wire/API/EnterpriseLogin.hs

```diff
@@ -93,10 +93,44 @@ domainRedirectSchema =
       NoneTag -> tag _None (pure ())
       LockedTag -> tag _Locked (pure ())
       SSOTag -> tag _SSO samlIdPIdObjectSchema
-      BackendTag -> tag _Backend backendUrlSchema
+      BackendTag -> tag (_Backend) backendConfigSchemaV8
       NoRegistrationTag -> tag _NoRegistration (pure ())
       PreAuthorizedTag -> tag _PreAuthorized (pure ())
 
+backendConfigSchemaV8 :: ObjectSchema SwaggerDoc (HttpsUrl, Maybe HttpsUrl)
+backendConfigSchemaV8 =
+  (,)
+    <$> fst .= backendUrlSchema
+    -- API versions <= V9 ignore the WebApp URL
+    <*> const Nothing .= pure Nothing
+
+domainRedirectSchemaV9 :: ObjectSchema SwaggerDoc DomainRedirect
```

Comment:
Maybe remove the v9 suffix, as this will be the schema for later versions as well.